nixos/modules/services/desktop-managers/cosmic.nix

```diff
@@ -47,9 +47,9 @@ in
 
   options = {
     services.desktopManager.cosmic = {
-      enable = lib.mkEnableOption "Enable the COSMIC desktop environment";
+      enable = lib.mkEnableOption "COSMIC desktop environment";
 
-      showExcludedPkgsWarning = lib.mkEnableOption "Disable the warning for excluding core packages." // {
+      showExcludedPkgsWarning = lib.mkEnableOption "the warning for excluding core packages." // {
```

Comment:
https://github.com/NixOS/nixpkgs/blob/31dd02fc7410cc97d71165d596f08d21bab7497f/pkgs/README.md?plain=1#L496

I interpret this to apply to option descriptions too

Reply:
Not exactly, but mkEnableOption already adds it, so removed it